examples/multi_agent_example.ipynb

```diff
@@ -0,0 +1,336 @@
+     ]
+    }
+   ],
+   "source": [
+    "ao_client = agentops.Client(AGENTOPS_API_KEY)\n",
+    "openai_client = openai.Client()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-03-08T20:31:50.318099Z",
+     "start_time": "2024-03-08T20:31:50.234493Z"
+    }
+   },
+   "id": "af062552554d60ce",
+   "execution_count": 3
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Now lets create a few agents!"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "95d212546aaf1f82"
+  },
+  {
+   "cell_type": "code",
+   "outputs": [],
+   "source": [
+    "@track_agent(name='qa')\n",
+    "class QaAgent:\n",
+    "    def completion(self, prompt: str):\n",
+    "        res = openai_client.chat.completions.create(model='gpt-3.5-turbo', messages=[{\"role\": \"system\", \"content\": \"You are a qa engineer and only output python code, no markdown tags.\"},\n",
+    "    {\"role\": \"user\", \"content\": prompt}], temperature=0.5)\n",
+    "        return res.choices[0].message.content\n",
+    "        \n",
+    "@track_agent(name='engineer')\n",
+    "class EngineerAgent:\n",
+    "    def completion(self, prompt: str):\n",
+    "        res = openai_client.chat.completions.create(model='gpt-3.5-turbo', messages=[{\"role\": \"system\", \"content\": \"You are a software engineer and only output python code, no markdown tags.\"},\n",
+    "    {\"role\": \"user\", \"content\": prompt}], temperature=0.5)\n",
+    "        return res.choices[0].message.content"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-03-08T20:31:50.318509Z",
+     "start_time": "2024-03-08T20:31:50.310816Z"
+    }
+   },
+   "id": "727e3cc26ce3ec3",
+   "execution_count": 4
+  },
+  {
+   "cell_type": "code",
+   "outputs": [],
+   "source": [
+    "qa = QaAgent()\n",
+    "engineer = EngineerAgent()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-03-08T20:31:50.375686Z",
+     "start_time": "2024-03-08T20:31:50.313794Z"
+    }
+   },
+   "id": "79b75d65de738522",
+   "execution_count": 5
+  },
+  {
+   "cell_type": "code",
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
+      "INFO:root:LLM call from agent named: engineer\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Unable to parse a chunk for LLM call {'model': 'gpt-3.5-turbo', 'messages': [{'role': 'system', 'content': 'You are a software engineer and only output python code, no markdown tags.'}, {'role': 'user', 'content': 'Write a python function that accepts two numbers and multiplies them together, then divides by two. No example.'}], 'temperature': 0.5} - skipping upload to AgentOps\n"
```

Comment:
You should probably clear the outputs and save?

Reply:
My thought was to leave outputs in so you can see the flow without running it. If we don't want to do that, I can clear it

Reply:
i cleared it and added more explanation in the text